iml-wire-types/src/snapshot.rs

```diff
@@ -131,6 +132,62 @@ impl FromStr for ReserveUnit {
     }
 }
 
+#[cfg_attr(feature = "graphql", derive(juniper::GraphQLObject))]
+#[derive(serde::Deserialize, serde::Serialize, Eq, Clone, Debug)]
+/// Automatic snapshot policy
+pub struct SnapshotPolicy {
```

Comment:
FYI, currently a weeks ends on Sunday. I think it would be the best to make it part of the policy, even if not visible for users.